src/test/addresses_mainnet.sol

```diff
@@ -46,6 +46,8 @@ contract Addresses {
         addr["MCD_FLIP_ETH_A"]         = 0xF32836B9E1f47a0515c6Ec431592D5EbC276407f;
         addr["MCD_JOIN_ETH_B"]         = 0x08638eF1A205bE6762A8b935F5da9b700Cf7322c;
         addr["MCD_FLIP_ETH_B"]         = 0xD499d71bE9e9E5D236A07ac562F7B6CeacCa624c;
+        addr["MCD_JOIN_ETH_C"]         = 0xF04a5cC80B1E94C69B48f5ee68a08CD2F09A7c3E;
+        addr["MCD_FLIP_ETH_C"]         = 0x7A67901A68243241EBf66beEB0e7b5395582BF17;
```

Comment:
The `beg` is wrong for this flip.

It's supposed to be 3%:https://forum.makerdao.com/t/eth-c-risk-parameters/6684
It is 5%: `seth call 0x7A67901A68243241EBf66beEB0e7b5395582BF17 'beg()' | seth --to-dec | seth --from-wei`

Reply:
That's the default that gets set in the constructor. The spell updates it to 3%.

Reply:
nevermind